versioned_docs/version-v1.3/upgrade/v1-3-1-to-v1-3-2.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 2
+sidebar_label: Upgrade from v1.3.1 to v1.3.2
+title: "Upgrade from v1.3.1 to v1.3.2"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/upgrade/v1-3-1-to-v1-3-2"/>
+</head>
+
+## General information
+
+An **Upgrade** button appears on the **Dashboard** screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).
+
+For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).
+
+
+## Known issues
+
+---
+
+### 1. Two-node cluster upgrade stuck after the first node is pre-drained
+
+:::info important
+
+It is advisable to shut down all the workload VMs before conducting the upgrade for **two-node** cluster upgrades.
```

Comment:
Shutting down the VMs is just a means to prevent potential data loss. This known issue itself is inevitable. We don't have a way to prevent it. Well, technically, there is a way, but that's infeasible for most users, and we don't want them to take the risks.

What's your suggestion for this paragraph according to the fact?

Reply:
Suggestion: Shut down all workload VMs before upgrading **two-node clusters** to prevent data loss.

*We should always try to explain the reasoning behind recommendations. Also, when severe consequences such as data loss and irreparable system damage are involved, preventative actions should be required instead of just recommended.*